src/shopify.js

```diff
@@ -0,0 +1,5 @@
+import Clevertap from './clevertap'
+
+const clevertap = new Clevertap({ mode: 'SHOPIFY' })
```

Comment:
The new `mode` options to control the behaviour of the SDK and change endpoints as well. This is not overridden by clients.